h5c/vic/src/vic-webapp/package-lock.json

```diff
@@ -0,0 +1,13461 @@
+{
```

Comment:
This seems unrelated to the change being made. Is this supposed to be included in this PR?

Reply:
yes, not related with this issue, but necessary